flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
```

Comment:
this is disappointing.

Reply:
Well, is it possible to download certain files and run them differently depending on the architecture because in that case it is possible to support `aarch64`

Reply:
like building from source?

Reply:
Well is it possible to run arch specific commands? If not maybe I can add `|| true`.